### PR TITLE
prettierx: add --break-before-else option

### PR DESCRIPTION
**Pull request by brody2consult**
as requested by many in <https://github.com/prettier/prettier/issues/840>

### CODE REVIEW

File: src/language-js/postprocess.js [hunk 1]
```diff
@@ -43,6 +43,10 @@ function postprocess(ast, options) {
           node.body.end = node.end - 1;
         }
         break;
+      case "IfStatement":
```
Comment (brody2consult) [inline src/language-js/postprocess.js]:
```suggestion
      case "IfStatement": // quick workaround for ast compare & breakBeforeElse
```

**Comment by brody2consult** [comment]
Recent test run failed on Node.js 14, seems to be due to an internal v8 error: https://github.com/brodybits/prettierx/pull/224/checks?check_run_id=613912851

restarting it now

**Comment by shannonmoeller** [comment]
If you're going for Stroustrup, "break before else" doesn't catch all cases. You also need to break before `catch`, `finally`, etc. Something like "break after block" would be closer to accurate.

**Comment by brody2consult** [comment]
> If you're going for Stroustrup, "break before else" doesn't catch all cases.

A new issue would be extremely helpful, especially if you can point to some popular examples just like Jake Niemiec did for the break before else in <https://github.com/prettier/prettier/issues/840#issuecomment-619260404>.

And a quick word of warning that I may be somewhat slow over the next few weeks due to an urgent work project. But yeah, flexibility should be an obvious goal of this project.

**Comment by jnunderwood** [comment]
Opened new issue to create option for Stroustrup braces (#237).

**Pull request merged**